paddle/operators/type_alias.h

```diff
@@ -0,0 +1,51 @@
+/* Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserve.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License. */
+
```

Comment:
Most operators are common about what they used. Just declare them in a common file.

If we declare them into every source file, it could be redefined error when some of them depends on each other. Like

```cpp
// op_a.h
using XXX = ...;
```

```cpp
#include "a.h"

using XXX = ...;  // redefined error.
```